dotcom-rendering/src/components/BylineLink.tsx

```diff
@@ -106,11 +110,17 @@ function removeComma(bylinePart: string) {
 		: bylinePart;
 }
 
-export const BylineLink = ({ byline, tags, format }: Props) => {
+export const BylineLink = ({
+	byline,
+	tags,
+	format,
+	isHeadline = false,
+}: Props) => {
 	const tokens = bylineAsTokens(byline, tags);
 	const soleContributor = getSoleContributor(tags, byline);
 	const hasSoleContributor = !!soleContributor;
 	const bylineComponents = getBylineComponentsFromTokens(tokens, tags);
+	const isLiveBlog = format.design === ArticleDesign.LiveBlog;
```

Comment:
Are we sure we do not want to also include dead blogs here?

Reply:
nope only live blogs as these have the pillar coloured background. dead blogs are just standard light/dark backgrounds